udev/50-cmsis-dap.rules

```diff
@@ -52,6 +52,9 @@ SUBSYSTEM=="usb", ATTR{idVendor}=="03eb", ATTR{idProduct}=="216c", MODE:="666"
 # 03eb:2175 Microchip nEDBG CMSIS-DAP
 SUBSYSTEM=="usb", ATTR{idVendor}=="03eb", ATTR{idProduct}=="2175", MODE:="666"
 
+# 0483:572a STMicroelectronics CMSIS-DAP
```

Comment:
The description should probably be changed to indicate that it's not an STMicro product. Maybe include a link to the firmware (since it doesn't have a very uniquely identifiable name)?

Reply:
Hope you don't mind, I went ahead and changed the description so I can merge the PR quickly.

Reply:
Thank you @flit!

I am currently traveling and might be slowly with the PR changes.

Reply:
No worries, all taken care of! 😄 Thanks for the PR.